docs/MigrationGuide.md

```
@@ -0,0 +1,1103 @@
+   ```promql
+   # Before (custom names)
+   rate(my_cache_hits[5m])
+   rate(my_cache_misses[5m])
+
+   # After (standard names)
+   rate(cache_hits_total{cache_name="main-cache"}[5m])
+   rate(cache_misses_total{cache_name="main-cache"}[5m])
+   ```
+
+4. **Preserve Historical Data**
+   ```promql
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)